components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -15,7 +16,7 @@ describe('d2l-list-item-drag-handle', () => {
 	let browser, page;
 
 	const focusMethod = (selector) => {
-		return page.$eval(selector, (item) => { item.focus(); });
+		return page.$eval(selector, (item) => { forceFocusVisible(item); });
```

Comment:
This was using `buttonStyles` which now doesn't have an outline for `:focus` so it needed to switch to this.